src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -12,17 +12,25 @@ public class NodeAutoCompleteSearchViewModel : SearchViewModel
     {
         internal PortViewModel PortViewModel { get; set; }
 
+        /// <summary>
+        /// Cache of default node suggestions, use it in case where
+        /// a. our algorithm does not return sufficient results
+        /// b. the results returned by our algorithm will not be useful for user
+        /// </summary>
+        internal IEnumerable<NodeSearchElementViewModel> DefaultResults { get; set; }
+
         internal NodeAutoCompleteSearchViewModel(DynamoViewModel dynamoViewModel) : base(dynamoViewModel)
         {
-            // Do nothing for now, but we may off load some time consuming operation here later
+            // Off load some time consuming operation here
+            InitializeDefaultAutoCompleteCandidates();
         }
 
         internal void InitializeDefaultAutoCompleteCandidates()
```

Comment:
I think this method can be private as `DefaultResults` is internal anway.

Reply:
yeah it was internal before we were calling it outside of this class